packages/ndla-ui/src/SearchTypeResult/SearchTypeHeader.tsx

```diff
@@ -77,39 +82,53 @@ export type FilterOptionsType = {
 };
 
 type Props = {
-  filters: FilterOptionsType[];
-  onFilterClick: (id: string) => void;
+  filters?: FilterOptionsType[];
+  onFilterClick?: (id: string) => void;
   totalCount: number;
   type?: ContentType;
 };
-const SearchTypeHeader = ({ filters, onFilterClick, totalCount, type, t }: Props & WithTranslation) => (
-  <HeaderWrapper>
-    <TypeWrapper>
-      {type && <ContentTypeBadge type={type} background size="large" />}
-      <SubjectName>
-        {type && <b>{t(`contentTypes.${type}`)}</b>}{' '}
-        {totalCount && <Count>{t(`searchPage.resultType.hits`, { count: totalCount })}</Count>}
-      </SubjectName>
-    </TypeWrapper>
-    {filters && (
+const SearchTypeHeader = ({ filters = [], onFilterClick, totalCount, type, t }: Props & WithTranslation) => (
```

Comment:
Eg trudde WithTranslation var for Components og ikkje funksjonelle komponenter. Men så lenge det fungerer skal ikkje eg krangle.

Reply:
Fiksa